tools/distro-scraper/scraper/cli.py

```diff
@@ -103,11 +103,16 @@ async def run_scraper(scraper_instance: BaseScraper) -> tuple[str, dict | None]:
 
     # Validate JSON structure
     try:
+        if result is None:
+            raise ValueError("No data returned from scraper")
+
```

Comment:
I think once we catch the exception on lines 101-102, we know for sure we won't be able to turn `result` into a `ScraperResult` (since it's guaranteed to be `None`). How about just returning early in that case?

```suggestion
        return name, None

    # Validate JSON structure
    try:
```